### ISSUE TITLE
transformer example missing the Transformer module?

### ISSUE THREAD

**Issue by mjfwest** [FabriceSalvaire/PySpice#10]
I don't think the transformer example is complete. The Transformer Module does not seem to be in the repository. 
``` Python
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
PySpice/examples/transformer/transformer.py in <module>()
     17 ####################################################################################################
     18 
---> 19 from Transformer import Transformer
     20 
     21 ####################################################################################################

ImportError: No module named 'Transformer'
```

**Comment by FabriceSalvaire** [comment]
Maybe I should tweak the sys.path ?

**Comment by mjfwest** [comment]
OK. I think I figured it out.  My directory is not case sensitive and when I check out the files I end up with either Transformer.py or transformer.py.  Could they be called something different for simplicity?

**Comment by FabriceSalvaire** [comment]
Oups it looks like a known Windows issue. Yes I have to rename.

**Comment by mjfwest** [comment]
I'm on OSX, and thought I was case sensitive...   Either way it is a fix for Windows.  With a bit more googling, I found it might be worth setting: 
`git config core.ignorecase true`

http://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git

**Comment by FabriceSalvaire** [comment]
fixed in commit d816fb4a6245ac2d2596693d2be1fa83538538d8
